plugins/modules/zos_blockinfile.py

```diff
@@ -185,6 +185,9 @@
     the block will be overwritten on each iteration.
   - When more then one block should be handled in a file you must change
     the I(marker) per task.
+  - If using ZOAU version 1.3.0 and double quotes (") in the block argument, the module
+    will throw a false negative response. Follow up on the
+    L(issue in the collection's repository,https://github.com/ansible-collections/ibm_zos_core/issues/1258).
```

Comment:
Perhaps the note here can be expanded upon to include some additional details. Maybe something like:
> With ZOAU version 1.3.0,  if double quotes (") are used in the I(block) parameter, the module
    fails with an error message containing: 'return content is NOT in json format'. This is currently a false negative response, where the I(src) is changed, but an error is reported anyhow. Follow up on the issue tracked L(here,https://github.com/ansible-collections/ibm_zos_core/issues/1258).

Feel free to re-reword what I've got, my point is just to add more info about the behavior for the user.